backend/internal/data/repo/repo_product_search.go

```diff
@@ -0,0 +1,20 @@
+package repo
+
+type BarcodeProduct struct {
+	SearchEngineName string `json:"search_engine_name"`
+
+	// Identifications
+	ModelNumber  string `json:"modelNumber"`
+	Manufacturer string `json:"manufacturer"`
+
+	// Extras
+	Country string `json:"notes"`
+	Barcode string `json:"barcode"`
+
+	// TODO: add image attachement
+	// TODO: add asin?
```

Comment:
These TODO comments are making it into the Swagger docs, I would recommend removing them or moving them to somwhere else in the file where the swagger generator won't pick them up.